packages/kbn-unified-field-list/src/components/field_item_drag_handle/field_item_drag_handle.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiIcon, useEuiTheme, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+export const FieldItemDragHandle: React.FC = () => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <EuiFlexGroup
+      className="unifiedFieldListItemDragHandle"
+      css={css`
+        pointer-events: none;
+        flex-shrink: 0;
```

Comment:
@jughosta do you still need this line? I'm seeing this in the browser.

<img width="410" alt="image" src="https://github.com/elastic/kibana/assets/4016496/72324b35-f2d8-4d91-92ec-7b2dbfddcc61">